settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'beats'
```

Comment:
I don't think we need this since there is only one project, I've found this file to be helpful for when there are submodules. what do you think? I am fine either way, mostly probing to hear opinion